### DISCUSSION TITLE
🔨iLogtail开发者社区月会（2024年5月22日 19:00-20:00）

### DISCUSSION THREAD

**Discussion by yyuuttaaoo** [alibaba/ilogtail#1493] [Announcements]
## 会议信息

时间：5月22日 周三 19:00 - 20:00 （GMT +8:00）
会议号：294695017
入会链接：https://meeting.dingtalk.com/j/g2yqevksyZR
可通过浏览器直接入会，无需下载钉钉，希望大家都能拨冗出席或事后观看视频。

### Part 1：社区最新动向（5min）

本次会介绍社区的最新动向，此外还有 _神秘_ 事项将会公布，敬请期待
- 社区人员架构更新
- 社区近期活动介绍
- _？？？_

### Part 2：社区分享（25min）

本次共有三名社区同学带来他们的分享：
- 【新功能】C++版Stdout采集功能上线（10min）[@quzard](https://github.com/quzard)
- 【最佳实践】iLogtail&SPL数据处理最佳实践（10min）[@Abingcbc](https://github.com/Abingcbc)
- 【案例分享】从开源版 iLogtail 到低成本混合存储日志查询服务（15min）[@liangry](https://github.com/liangry)

### Part 3：开发问答（20min）

为了支持各位社区开发者的开发工作，我们新增了一个开发问答环节，您可以提前在本Discussion下留言或现场提问关于iLogtail开发过程遇到的困惑、问题，我们会在会议上交流讨论，分享开发经验和心得。

**Comment by youaodu** [comment]
麻烦问下，会议一般多久举行一次，怎么参加，有线上会议吗。

**Reply by Takuka0311** [reply]
一般1～2月会举行一次，最近的一次在本月末，届时会发布通知

**Reply by Takuka0311** [reply]
参会方式都会随通知发布，都是线上会议
